storybook/src/pages/amsterdam/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,111 @@
+                <Label htmlFor="body">Wat wilt u aan de gemeente vragen?</Label>
+                <Paragraph id="bodyDescription" size="small">
+                  Een duidelijke beschrijving van uw vraag helpt ons bij het behandelen.
+                </Paragraph>
+                <TextArea
+                  aria-describedby="bodyDescription"
+                  id="body"
+                  onChange={(e) => setTextareaLength(e.target.value.length)}
+                  rows={4}
+                />
+                <FormFieldCharacterCounter length={textareaLength} maxLength={1000} />
+              </Field>
+              <FieldSet aria-describedby="contactDetailsDescription" legend="Wat zijn uw gegevens?">
+                <Column gap="small">
+                  <Paragraph id="contactDetailsDescription">
+                    Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.
+                  </Paragraph>
+                  <Field>
+                    <Label htmlFor="initials">Voorletters</Label>
+                    <TextInput id="initials" name="initials" />
+                  </Field>
+                  <Field>
+                    <Label htmlFor="familyName">Achternaam</Label>
+                    <TextInput autoComplete="family-name" id="familyName" name="familyName" />
+                  </Field>
+                  <FieldSet legend="Woonplaats">
+                    <Radio name="city" value="amsterdam">
+                      Amsterdam
+                    </Radio>
+                    <Radio name="city" value="weesp">
+                      Weesp
+                    </Radio>
+                    <Radio name="city" value="anders">
+                      Anders
+                    </Radio>
+                  </FieldSet>
+                  <Field>
+                    <Label htmlFor="email">E-mail</Label>
+                    <TextInput autoComplete="email" id="email" name="email" />
+                  </Field>
+                  <Row>
+                    <Field>
+                      <Label htmlFor="countryCode">Landnummer</Label>
+                      <Select name="countryCode">
+                        <option value="+31">Nederland +31</option>
+                        <option value="+32">België +32</option>
+                        <option value="+33">Frankrijk +33</option>
+                      </Select>
+                    </Field>
+                    <Field>
+                      <Label htmlFor="phone">Telefoonnummer</Label>
+                      <TextInput autoComplete="phone" id="phone" name="phone" />
+                    </Field>
+                  </Row>
+                </Column>
+              </FieldSet>
+              <Alert severity="info">
+                <Paragraph>
+                  We bewaren uw contactgegevens voor het afhandelen van uw vraag of klacht en het verbeteren van onze
+                  dienstverlening.
+                </Paragraph>
+              </Alert>
+              <div>
```

Comment:
Perhaps a `Row` is recommended to anticipate multiple buttons with some spacing.